src/localization/dev_best_practices.md

```diff
@@ -383,6 +383,14 @@ Another good example is *Yes/No*. There are two types of languages, those that h
 
 W3C has a good [guide](http://www.w3.org/International/articles/article-text-size) on the length ratios a developer should be prepared for.
 
+### Design to avoiding breaking words
+
+Not only do strings become longer (as above), individual words can be much longer than you expect in some languages. This could lead to strings overflowing, particularly if a design contains elements with small widths or uses a large font. For example:
+
+* `Account recovery key` in English becomes `Kontowiederherstellungsschlüssel` in German.
```

Comment:
We can avoid the bullet point, since it's only one example, and put if after the colon.